contracts/BSCGovernor.sol

```diff
@@ -9,13 +9,13 @@ import "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorTimelo
 import "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorVotesQuorumFractionUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/governance/extensions/GovernorPreventLateQuorumUpgradeable.sol";
 
-import "./System.sol";
-import "./extension/Protectable.sol";
-import "./lib/Utils.sol";
-import "./interface/IGovToken.sol";
+import "./SystemV2.sol";
+import "./BC_fusion/extension/Protectable.sol";
```

Comment:
can we move the extension directory under `BC_fusion` outside as well?

Reply:
also can we move the fils that no longer used to a directory

Reply:
> a



> also can we move the fils that no longer used to a directory

ok, will move these files

Reply:
> can we move the extension directory under `BC_fusion` outside as well?

This will cause the root directory to be too bloated